libs/components/src/lib/popup/popup.spec.ts

```diff
@@ -156,10 +156,48 @@ describe('vwc-popup', () => {
 			expect(element.dismissible).toBeFalsy();
 			expect(element.anchor).toBeUndefined();
 			expect(element.placement).toBeUndefined();
+			expect(element.placementStrategy).toBe(PlacementStrategy.Flip);
+			expect(element.animationFrame).toBe(false);
 			expect(element.strategy).toEqual('fixed');
 		});
 	});
 
+	describe('open', () => {
+		beforeEach(() => {
+			jest.spyOn(floatingUI, 'autoUpdate');
+		});
+
+		afterEach(() => {
+			jest.mocked(floatingUI.autoUpdate).mockRestore();
+		});
+
+		it('should hide control if not set', async () => {
+			expect(getControlElement(element).classList).not.toContain('open');
+		});
+
+		it('should show control if set', async () => {
+			element.open = true;
+			await elementUpdated(element);
+
+			expect(getControlElement(element).classList).toContain('open');
+		});
+
+		it('should not begin to auto update position before DOM is updated', async function () {
+			element.anchor = anchor;
+			element.open = true;
+
+			expect(floatingUI.autoUpdate).not.toHaveBeenCalled();
+		});
+
+		it('should begin to auto update after DOM is updated', async function () {
+			element.anchor = anchor;
+			element.open = true;
+			await elementUpdated(element);
+
+			expect(floatingUI.autoUpdate).toHaveBeenCalled();
+		});
```

Comment:
These can be combined to a single test:
```suggestion
		it('should begin to auto update after DOM is updated', async function () {
			element.anchor = anchor;
			element.open = true;
			const updateCallsBeforeDOMUpdate = floatingUI.autoUpdate.mock.calls.length;
			await elementUpdated(element);
      expect(updateCallsBeforeDOMUpdate).toBe(0);
			expect(floatingUI.autoUpdate).toHaveBeenCalledTimes(1);
		});
```